src/openapi.yaml

```diff
@@ -3311,6 +3311,32 @@ paths:
               $ref: '#/components/headers/X-Is-Form-Designer'
             Content-Language:
               $ref: '#/components/headers/Content-Language'
+  /api/v2/services/:
```

Comment:
I considered adding it under the config. Because that's roughly where it lives in the admin interface. But moved it here: flat is better than nested.


TODO this endpoint should be flagged private/internal/subject to change. But I couldn't find out how to do that.

Reply:
there is a `openforms.api.utils.mark_experimental` decorator :-)